lib/foreman_remote_execution_core.rb

```diff
@@ -34,7 +34,7 @@ def self.validate_ssh_log_level!
 
     current = if defined?(::Proxy::SETTINGS)
                 ::Proxy::SETTINGS.log_level.to_s.downcase
-              elsif defined?(SmartProxyDynflowCore)
+              elsif defined?(SmartProxyDynflowCore::SETTINGS)
```

Comment:
would it make sense to do rather ` defined?(SmartProxyDynflowCore) && defined?(SmartProxyDynflowCore::SETTINGS)`

Reply:
Is it necessary? What would be the scenario where your suggestion would behave differently?

```
2.7.0 :001 > defined?(::SmartProxyDynflowCore)
 => nil 
2.7.0 :002 > defined?(::SmartProxyDynflowCore::SETTINGS)
 => nil
```

Reply:
@ares ping?

Reply:
ah true, ignore me